orttraining/orttraining/training_ops/cuda/optimizer/common.h

```diff
@@ -3,11 +3,21 @@
 
 #pragma once
 #include "core/providers/cuda/cuda_common.h"
-#include "orttraining/training_ops/cpu/optimizer/common.h"
 
 namespace onnxruntime {
 namespace cuda {
 
+template <typename TC, typename TS>
+TC compute_bias_correction_coefficient(
```

Comment:
why duplicate this? why not use the one in onnxruntime::contrib?

Reply:
You are right. Removed this duplication in cuda path